spec/support/refunds_steps/setup_steps.rb

```diff
@@ -7,7 +7,7 @@ def given_i_am_luke_skywalker
   self.test_user = OpenStruct.new title: "Mr",
                                   first_name: "Luke",
                                   last_name: "Skywalker",
```

Comment:
Not sure you need this ?  This is in the 'refunds' part of the application which you are not touching.